tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./node_modules/aegir/src/config/tsconfig.aegir.json",
```

Comment:
Let's not assume where exactly aegir is.

```suggestion
  "extends": "aegir/src/config/tsconfig.aegir.json",
```

Reply:
That started failing recently and we don't know why yet.

Reply:
The typescript team broke this a minor release last week:

```
> is-ipfs@3.0.0 prepare
> aegir build --no-bundle

[11:11:07] Clean ./dist [started]
[11:11:07] Clean ./dist [completed]
[11:11:07] Generate types [started]
tsconfig-types.aegir.json:1:12 - error TS6053: File 'aegir/src/config/tsconfig.aegir.json' not found.
```